doc/denite.txt

```diff
@@ -254,6 +254,10 @@ jump_point
 						*denite-source-grep*
 grep		Gather grep results and nominates them.
 
+						*denite-source-filetype*
```

Comment:
let me know if you want to sort the sources. This ways as more sources are added it will be easier for users to find what they want.